python/pyhie/allspark/model/qwen_v15.py

```
@@ -546,33 +615,22 @@ def _trans_weight(self, torch_weight, lora_name=None):
                                 f"Weight Tensor Name: {key}, Split Mode: HSPLIT, Shape: {tensor.shape}"
                             )
                     if (group_list!=[] and self.quant_config.quantize_mode == QuantizeConfig.QuantMode.A16W4):
-                        save_torch_to_allsparky(weights_path, key, qdata, mode,
-                                            split_mode, [x//2 for x in group_list ])
+                        _w(key, qdata, mode, split_mode, [x//2 for x in group_list])
                     else:
-                        save_torch_to_allsparky(weights_path, key, qdata, mode,
-                                                split_mode, group_list)
+                        _w(key, qdata, mode, split_mode, group_list)
                     if (self.quant_config.extra_option["SubChannel"] == True or split_mode != HSPLIT) and self.quant_config.quantize_mode != QuantizeConfig.QuantMode.FP8A8W8:
```

Flake8 (7.3.0)

[error] comparison to True should be 'if cond is True:' or 'if cond:'

(E712)